CHANGELOG.md

```diff
@@ -4,6 +4,146 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.26.0] - 2022-01-24
+
+This release adds a number of improvements, most notably:
+
+- We make Substrate dependencies optional ([#760](https://github.com/paritytech/subxt/pull/760)), which makes WASM builds both smaller and more reliable. To do this, we re-implement some core types like `AccountId32`, `MultiAddress` and `MultiSignature` internally.
+- Allow access to storage entries ([#774](https://github.com/paritytech/subxt/pull/774)) and runtime API's ([#777](https://github.com/paritytech/subxt/pull/777)) from some block. This is part of a move towards a more "block centric" interface, which will better align with the newly available `chainHead` style RPC interface.
+- Add RPC methods for the new `chainHead` style interface. These are currently unstable, but will allow users to start experimenting with this new API if their nodes support it.
```

Comment:
I think you could add a link to the RPC v2 spec, when you state `new API` this could be quite hard to understand that you refer to https://paritytech.github.io/json-rpc-interface-spec/ :)

Reply:
Good point!